src/main/java/com/liefery/android/stop_badge/StopBadge.java

```diff
@@ -236,7 +246,19 @@ int shadowSizeY() {
         return (int) ( shadowRadius + Math.abs( shadowDx ) );
     }
 
-    public Bitmap export( int size ) {
+    public String toKey( int size ) {
+        StringBuilder builder = new StringBuilder();
+        builder.append( pathContents );
```

Comment:
Could you try if the Java default Path.hashCode method generates useful values for the Path objects?